src/Cli/OptionCommand.php

```diff
@@ -0,0 +1,383 @@
+		}
+
+		$options = $this->get_options();
+
+		if ( $options instanceof WP_Error ) {
+			/* translators: %s: error message */
+			WP_CLI::error( sprintf( __( 'Could not retrieve options: %s', 'amp' ), $options->get_error_message() ) );
+		}
+
+		if ( ! isset( $options[ $option_name ] ) ) {
+			/* translators: %s: option name */
+			WP_CLI::error( sprintf( __( 'Could not get %s option. Does it exist?', 'amp' ), $option_name ) );
+		}
+
+		WP_CLI::print_value( $options[ $option_name ], $assoc_args );
+	}
+
+	/**
+	 * Updates an option value.
+	 *
+	 * ## OPTIONS
+	 *
+	 * <key>
+	 * : The name of the option to update.
+	 *
+	 * <value>
+	 * : The new value.
+	 *
+	 * ## EXAMPLES
+	 *
+	 * # Update plugin option.
+	 * $ wp amp option update theme_support reader
+	 * Success: Updated theme_support option.
+	 *
+	 * @alias set
+	 *
+	 * @param array $args       Array of positional arguments.
+	 * @param array $assoc_args Associative array of associative arguments.
+	 */
+	public function update( $args, $assoc_args ) {
+		list( $option_name, $option_value ) = $args;
+
+		$user_cap = $this->check_user_cap();
+
+		if ( $user_cap instanceof WP_Error ) {
+			WP_CLI::error( $user_cap->get_error_message( 'amp_rest_cannot_manage_options' ) . PHP_EOL . WP_CLI::colorize( '%y' . $user_cap->get_error_message( 'amp_rest_cannot_manage_options_help' ) . '%n' ) );
+		}
+
+		$options = $this->get_options();
+
+		if ( $options instanceof WP_Error ) {
+			/* translators: %s: error message */
+			WP_CLI::error( sprintf( __( 'Could not retrieve options: %s', 'amp' ), $options->get_error_message() ) );
+		}
+
+		if ( ! isset( $options[ $option_name ] ) ) {
+			/* translators: %s: option name */
+			WP_CLI::error( sprintf( __( 'Could not update %s option. Does it exist?', 'amp' ), $option_name ) );
+		}
+
+		if ( ! in_array( $option_name, self::ALLOWED_OPTIONS, true ) ) {
+			/* translators: %s: option name */
+			WP_CLI::error( sprintf( __( 'You are not allowed to update %s option via the CLI.', 'amp' ), $option_name ) );
+		}
+
+		$update = $this->update_option( $option_name, $option_value );
+
+		if ( $update instanceof WP_Error ) {
+			/* translators: %1$s: option name, %2$s: error message */
+			WP_CLI::error( sprintf( __( 'Could not update %1$s option: %2$s', 'amp' ), $option_name, $update->get_error_message() ) );
+		}
+
+		/* translators: %s: option name */
+		WP_CLI::success( sprintf( __( 'Updated %s option.', 'amp' ), $option_name ) );
+	}
+
+	/**
+	 * Lists options and their values.
+	 *
+	 * This command can be used to list following options:
+	 * - AMP Plugin options.
+	 * - AMP Plugin options that can be updated via the CLI.
+	 * - Reader themes.
+	 *
+	 * ## OPTIONS
+	 * [<cli-managed-options>]
+	 * : List of options that can be updated via the CLI.
+	 *
+	 * [<reader-themes>]
+	 * : List of Reader themes.
```

Comment:
IMO, it may help with discoverability and simplify the code a bit if we make the reader theme list a sibling of the `list` command. So like this:

```
wp amp option list
wp amp option list-reader-themes
```

Then there won't be the need to check the `$args` and have custom command routing.